pylint/checkers/base.py

```diff
@@ -2075,6 +2147,68 @@ def _name_became_keyword_in_version(name, rules):
                 return ".".join(str(v) for v in version)
         return None
 
+    @staticmethod
+    def _assigns_typevar(node: Optional[nodes.NodeNG]) -> bool:
+        """Check if a node is assigning a TypeVar."""
+        if isinstance(node, astroid.Call):
+            inferred = utils.safe_infer(node.func)
+            if (
+                inferred
+                and isinstance(inferred, astroid.ClassDef)
+                and inferred.qname() == TYPING_TYPE_VAR_QNAME
+            ):
+                return True
+        return False
+
+    @staticmethod
+    def _check_typevar_variance(
+        name: str, node: nodes.AssignName
+    ) -> List[Tuple[str, str, str]]:
+        """Check if a TypeVar has a variance and if it's included in the name.
+
+        Returns the args for the message to be displayed.
+        """
+        args_to_display: List[Tuple[str, str, str]] = []
+
+        if isinstance(node.parent, nodes.Assign):
+            keywords = node.assign_type().value.keywords
+        elif isinstance(node.parent, nodes.Tuple):
+            keywords = (
+                node.assign_type().value.elts[node.parent.elts.index(node)].keywords
+            )
+        else:
+            return args_to_display
```

Comment:
This line is not tested either.

Reply:
https://github.com/PyCQA/pylint/pull/5221#discussion_r739526643

😄 It was actually you who suggested to add this. But removed in latest commit.

Reply:
Haha sorry. But yeah if we can't find a way to cover it then it's safe to remove :)